mail_suggested_recipient_unchecked/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+#. Go to Contacts and create a new one with an email address.
```

Comment:
This is still a specific case, although included in the `base` module. The idea is to redact a generic text like "#. On a chatter where recipients can be suggested, click on "Send Message", and bla, bla, bla", and then you can put an example where it happens, like the contact one, but don't make users thing that this is only working for that case.

Reply:
Now it's the example what is missing, hehe.

Reply:
Changed, is it correct now?